data/base-spells.yaml

```diff
@@ -17,6 +17,7 @@ invoke_messages:
 - Kneeling down, you draw the spell pattern's shadow with
 - You make sweeping gestures through the air
 - You lift your staff reverently
+- You lift your staff toward the .* and will the mana streams to bend into it
```

Comment:
I recommend combining this with line 19, maybe `You lift your staff (reverently|toward)`?